test/unit/math/mix/fun/add_test.cpp

```diff
@@ -106,6 +106,14 @@ TEST(MathMixMatFun, add_mat_mat) {
   stan::test::expect_ad(f, m23, m23b);
 }
 
+TEST(MathMixMatFun, add_scal) {
+  auto f = [](const auto& x, const auto& y) { return stan::math::add(x, y); };
+
+  double a = 2;
```

Comment:
There's no reason to define variables `a` and `b` here.  It also highlights that we want integer tests.  So this should be:

```
expect_ad(f, 2, 5);
expect_ad(f, 2.0, 5);
expect_ad(f, 2, 5.0);
expect_ad(f, 2.0, 5.0);
```
which should test all the instantiations work properly.